module-4/remove_metadata/remove_metadata.py

```diff
@@ -0,0 +1,46 @@
+import os
+import subprocess
+import argparse
+
+def remove_metadata_exiftool(input_file, output_file):
+    """
+    Removes metadata using ExifTool.
+    """
```

Comment:
docstring misses parameter information and info about return value of the function